src/pages/workspaces/workspace/Data.js

```diff
@@ -740,12 +769,27 @@ const WorkspaceData = _.flow(
               retryFunction: loadEntityMetadata
             }, [
               _.some({ targetWorkspace: { namespace, name } }, asyncImportJobs) && h(DataImportPlaceholder),
+              isSearchAwesomeNow() && !_.isEmpty(sortedEntityPairs) && div({ style: { margin: '1rem' } }, [
+                h(ConfirmedSearchInput, {
+                  'aria-label': 'Search all tables',
+                  placeholder: 'Search all tables',
+                  onChange: activeCrossTableTextFilter => {
+                    setActiveCrossTableTextFilter(activeCrossTableTextFilter)
+                    searchAcrossTables(_.keys(entityMetadata), activeCrossTableTextFilter)
+                  },
+                  defaultValue: activeCrossTableTextFilter
+                })
+              ]),
+              activeCrossTableTextFilter !== '' && div({ style: { margin: '0rem 1rem 1rem 1rem' } }, crossTableSearchInProgress ? ['Loading...', [icon('loadingSpinner', { size: 13, color: colors.primary() })]] : [`${_.sum(_.map(c => c.filteredCount, crossTableResultCounts))} results`]),
```

Comment:
The icon here is wrapped in an extra array. This results in a warning from React: `Warning: Each child in a list should have a unique "key" prop.`

```suggestion
              activeCrossTableTextFilter !== '' && div({ style: { margin: '0rem 1rem 1rem 1rem' } }, crossTableSearchInProgress ? ['Loading...', icon('loadingSpinner', { size: 13, color: colors.primary() })] : [`${_.sum(_.map(c => c.filteredCount, crossTableResultCounts))} results`]),
```